observability/micrometer/src/test/java/org/wildfly/extension/micrometer/SubsystemParsingTestCase.java

```diff
@@ -24,11 +23,7 @@ public static Iterable<MicrometerSubsystemSchema> parameters() {
     }
 
     public SubsystemParsingTestCase(MicrometerSubsystemSchema schema) {
-        super(MicrometerConfigurationConstants.NAME, new MicrometerExtension(), schema, MicrometerSubsystemSchema.CURRENT);
-    }
-
-    @Override
-    protected String getSubsystemXmlPathPattern() {
-        return "%s_%d_%d.xml";
+        super(MicrometerConfigurationConstants.NAME, new MicrometerExtension(), schema,
+            MicrometerSubsystemSchema.VERSION_2_0_COMMUNITY);
```

Comment:
This is not correct. The "current" schema will vary depending on the stability assumed by the test.

Reply:
@pferraro How would one modify these types of tests to be Stability-aware? Parameterized over the the different levels?